platform/Dockerfiles/cloud-hypervisor/build.sh

```diff
@@ -1,4 +1,6 @@
+CLOUD_HYPERVISOR_VERSION
 date=$(date '+%Y%m%d%H%M%S')
 
-docker buildx build --tag artificialwisdomai/cloud-hypervisor:v39.0-${date} --output type=docker --progress plain .
-docker buildx build --tag artificialwisdomai/cloud-hypervisor:v39.0-${date} --output type=local,dest="${PWD}/target" --progress plain .
+docker buildx build --tag artificialwisdomai/cloud-hypervisor:${CLOUD_HYPERVISOR_VERSOIN}-${date} --output type=docker --progress plain .
```

Comment:
The variable is not correctly spelled here.

Reply:
thanks @MostAwesomeDude .